app/code/Magento/CatalogGraphQl/Model/Resolver/Product/ProductHtmlAttribute.php

```diff
@@ -20,53 +19,38 @@
  */
 class ProductHtmlAttribute implements ResolverInterface
 {
-    /**
-     * @var ValueFactory
-     */
-    private $valueFactory;
-
     /**
      * @var OutputHelper
      */
     private $outputHelper;
 
     /**
-     * @param ValueFactory $valueFactory
      * @param OutputHelper $outputHelper
      */
     public function __construct(
-        ValueFactory $valueFactory,
         OutputHelper $outputHelper
     ) {
-        $this->valueFactory = $valueFactory;
         $this->outputHelper = $outputHelper;
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function resolve(
         Field $field,
         $context,
         ResolveInfo $info,
         array $value = null,
         array $args = null
-    ): Value {
+    ) {
         if (!isset($value['model'])) {
-            $result = function () {
-                return null;
-            };
-            return $this->valueFactory->create($result);
+            throw new GraphQlInputException(__('"model" value should be specified'));
```

Comment:
Should be `\LogicException`. Related discussion https://github.com/magento/graphql-ce/pull/132/files/a4c941e5fa7215e036198bd61c241434c03036ac..d86087487c0d505400c82d7e18dc6e0d6238464c

Reply:
https://github.com/magento/graphql-ce/issues/189
Has been created